hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/GetPartitionsMessageTask.java

```diff
@@ -61,7 +62,7 @@ protected ClientMessage call() {
             Address owner = partitions[i].getOwnerOrNull();
             int index = -1;
             if (owner == null) {
-                return null;
+                return VoidResultParameters.encode();
```

Comment:
This is what you suggested when returning null, I've no idea. just show me the correct version here.